### PR TITLE
fix readme for psenet inference

### PR DESCRIPTION
**Pull request by liangxhao**
Thank you for your contribution to the MindOCR repo.
Before submitting this PR, please make sure:

- [x] You have read the [Contributing Guidelines on pull requests](https://github.com/mindspore-lab/mindcv/blob/main/CONTRIBUTING.md)
- [x] Your code builds clean without any errors or warnings
- [x] You are using approved terminology
- [ ] You have added unit tests

## Motivation

更新PSENet的推理Readme

提供个样例，其它模型的Readme都按照这种格式来更新。

## Test Plan

(How should this PR be tested? Do you require special setup to run the test or repro the fixed bug?)

## Related Issues and PRs

(Is this PR part of a group of changes? Link the other relevant PRs and Issues here. Use https://help.github.com/en/articles/closing-issues-using-keywords for help on GitHub syntax)

### CODE REVIEW

File: configs/det/psenet/README_CN.md [hunk 1]
```diff
@@ -152,45 +152,43 @@ python tools/eval.py --config configs/det/psenet/pse_r152_icdar15.yaml
...
-完成output.mindir文件导出后，在/mindocr/deploy/py_infer目录下使用以下命令即可进行推理和评估：
+其中，`data_shape`是导出MindIR文件的模型输入值height和width。
+
+- 环境搭建
+
+请参考[环境安装](../../../docs/cn/inference/environment_cn.md#2-mindspore-lite推理)教程，配置MindSpore Lite推理运行环境。
+
+- 模型转换
+
+请参考[模型转换](../../../docs/cn/inference/convert_tutorial_cn.md#1-mindocr模型)教程，使用`converter_lite`工具对MindIR模型进行离线转换，其中`configFile`文件中的`input_shape`需要填写模型导出时shape，如上述的(1,3,1472,2624)。
```
Comment (HaoyangLee) [inline configs/det/psenet/README_CN.md:175]: 可补充说明该shape是NCHW

File: configs/det/psenet/README_CN.md [hunk 2]
```diff
@@ -152,45 +152,43 @@ python tools/eval.py --config configs/det/psenet/pse_r152_icdar15.yaml
 
 ### 3.6 MindSpore Lite 推理
 
-在进行推理前，请确保PSENet的后处理部分已编译（参考训练章节的后处理部分），并完成[推理环境搭建](../../../docs/cn/inference/environment_cn.md)。完成上述步骤后，请先下载[MindIR模型](https://download.mindspore.cn/toolkits/mindocr/psenet/psenet_resnet152_ic15-6058a798-0d755205.mindir)文件或使用以下命令将已训练完成的ckpt导出成MindIR文件:
```
Comment (VictorHe-1) [inline configs/det/psenet/README_CN.md:155]: 还是需要强调下PSENet的后处理部分需要先编译完成，才能进行推理

File: configs/det/psenet/README_CN.md (hunk 1 above)
Comment (liangxhao) [reply configs/det/psenet/README_CN.md]: 已修改

File: configs/det/psenet/README_CN.md (hunk 2 above)
Comment (liangxhao) [reply configs/det/psenet/README_CN.md:155]: 已修改

**Review by HaoyangLee — APPROVED** [review]

**Review by VictorHe-1 — APPROVED** [review]

**Pull request merged**